### PR TITLE
COMMERCE-9800

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://liferay.atlassian.net/browse/COMMERCE-9800

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase [hunk 1]
```diff
@@ -15,6 +15,59 @@ definition {
 		CommerceConfiguration.commerceTearDown();
 	}
 
+	@description = "This is a test for COMMERCE-9800. User can add a sample ULR in the virtual product's after the select file is removed"
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase:18]: "This is a test for COMMERCE-9800. Users can add a sample URL to the virtual products after the selected file is removed."

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase [hunk 2]
```diff
@@ -15,6 +15,59 @@ definition {
...
+	@priority = 3
+	test CanAddSampleURLAfterFileIsRemoved {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase:20]: CanAddSampleURLAfterRemovingSelectedFile

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase [hunk 3]
```diff
@@ -15,6 +15,59 @@ definition {
...
+
+			CommerceEntry.gotoMenuTab(menuTab = "Virtual");
+
+			CommerceProducts.updateProductVirtualConfiguration(
+				enableSample = "true",
+				sampleFileName = "Commerce_White.jpg",
+				virtualProductFile = "Commerce_Blue.jpg");
+		}
+
+		task ("When the user remove the file") {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase:52]: "removes"

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase [hunk 4]
```diff
@@ -15,6 +15,59 @@ definition {
...
+			Click(
+				key_sampleFileName = "Commerce_White.jpg",
+				locator1 = "CommerceEntry#DELETE_SAMPLE_FILE_ATTACHMENT");
+		}
+
+		task ("Then the user can add a sample filr URL") {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase:58]: Then a sample file URL can be added

**Review by baharturk — CHANGES_REQUESTED** [review]
Please take a look at my comments. Thanks!

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Review by baharturk — APPROVED** [review]

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
Issues are not related to the changes, PR can be forward

@victoroscuro

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**